HW8/marian_hrychak/task_3/math_utils/__init__.py

```
@@ -0,0 +1,3 @@
+from .area_figures import *
```

Ruff (0.8.2)

`from .area_figures import *` used; unable to detect undefined names

(F403)